pkgs/tools/nix/nixpkgs-fmt/default.nix

```
@@ -1,16 +1,16 @@
 { lib, rustPlatform, fetchFromGitHub, fetchpatch }:
```

```suggestion
{ lib, rustPlatform, fetchFromGitHub }:
```

See https://github.com/NixOS/nixpkgs/pull/113939#issuecomment-782908253